Testing/SystemTests/tests/qt/SliceViewerIntegrationTest.py

```diff
@@ -488,6 +488,31 @@ def runTest(self):
         pres.view.close()
 
 
+class SliceViewerTestAxesLimitsWithLogMatrixTransform(systemtesting.MantidSystemTest, HelperTestingClass):
```

Comment:
This is a good test, but I'm wondering if it really is an integration test. For example the model unit tests 
`qt/python/mantidqt/mantidqt/widgets/sliceviewer/test/test_sliceviewer_model.py`
have tests such as
`test_calculate_axes_angles_uses_identity_if_W_unavailable` 
Also I still don't think we have a test to defualt to `eye(3) when the basis vectors are 0 and the log is unavailable (apologies if you are in the middle of writing it already!)

Reply:
I removed the integration test and added the corresponding tests to `test_calculate_axes_angles_uses_identity_if_W_unavailable` and `test_calculate_axes_angles_uses_identity_if_W_available`

Reply:
Thanks, this is the last change request I promise! 
I think they should have gone in their own tests e.g. 
`test_calculate_axes_angles_uses_identity_if_W_unavailable_MDHisto`
I wouldn't normally ask you to change anythingn jsut for that, but I think it would be good to do an assertion on the `proj_matrix` stored in the model (as you originally did in the integration test) as well as the angles (I think I wrote those tests before I split up the proj_matrix and angle calculating code into separate functions!).

Reply:
Good suggestion! I split the tests.